java/vector/src/main/codegen/templates/UnionVector.java

```diff
@@ -550,15 +532,5 @@ public void setType(int index, MinorType type) {
     }
 
     @Override
-    public void reset() { }
-
-    @Override
-    public void generateTestData(int values) { }
-  }
-
-  public int getValueCount() { return 0; }
-
-  public void setValueCount(int valueCount) { }
-
-  public Object getObject(int index) { return null; }
+    public int getNullCount() { return 0; }
```

Comment:
Hmm.. Why is this returning 0?

Reply:
As explained here -- https://github.com/apache/arrow/pull/1203#discussion_r146049393

Reply:
Should this be `UnsupportedOperationException`?

Reply:
This is resolved.